test-app/package.json

```diff
@@ -33,19 +33,19 @@
   },
   "devDependencies": {
     "@babel/core": "^7.27.1",
-    "@ember/optional-features": "^2.0.0",
+    "@ember/optional-features": "^2.2.0",
     "@ember/string": "^3.1.1",
     "@ember/test-helpers": "^2.9.3",
     "@ember/test-waiters": "^4.1.0",
     "@embroider/macros": "^1.17.2",
     "@embroider/test-setup": "^4.0.0",
-    "@glimmer/component": "^1.0.4",
-    "@glimmer/tracking": "^1.0.4",
+    "@glimmer/component": "^2.0.0",
+    "@glimmer/tracking": "^1.1.2",
     "active-model-adapter": "^3.0.1",
     "ember-auto-import": "^2.10.0",
-    "ember-cli": "~3.28.0",
-    "ember-cli-babel": "^7.26.6",
-    "ember-cli-htmlbars": "^5.7.1",
+    "ember-cli": "~4.12.0",
+    "ember-cli-babel": "^8.2.0",
+    "ember-cli-htmlbars": "^6.3.0",
```

Comment:
just bumping stuff to latest that aren't peerdep contracts (where for those we need to test lower versions)